### PR TITLE
Added Resources

### PR DESCRIPTION
**Pull request by MichaelCade**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ Kubestr can assist in the following ways-
...
 
+## Resources 
+Video 
+* [Cloud Native Live: Introducing Kubestr – A New Way to Explore your Kubernetes Storage Options](https://youtu.be/N79NY_0aO0w)
+* [Introducing Kubestr - A handy tool for Kubernetes Storage](https://youtu.be/U3Rt9vcuQdc)
+* [A new way to benchmark your kubernetes storage DoK Talks #71](https://www.youtube.com/watch?v=g64eIOk_Ob4)
+
+
+Blogs 
+* [Benchmarking and Evaluating Your Kubernetes Storage with Kubestr](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjpp6nn4NDyAhWXiVwKHaxHBigQFnoECAMQAQ&url=https%3A%2F%2Fblog.kasten.io%2Fbenchmarking-kubernetes-storage-with-kubestr&usg=AOvVaw1iH_P27EO2waz81znslF_Q)
```
Comment (maakuth) [inline README.md:24]: This URL appears funny, maybe it is a copy from a search result page instead of the proper page URL?

**Review by MichaelCade — COMMENTED** [review]
Updated URL

**Comment by saiyam1814** [comment]
I see there is this resource PR , wanted to add a resource for kubestr as well, what do you suggest @MichaelCade ?

**Comment by MichaelCade** [comment]
Hey, @saiyam1814 Can you see the commits made in this PR? if you add in your awesome demo there we can get this also added to help new users.

**Comment by julio-lopez** [comment]
@bathina2 @MichaelCade Should we merge this one and have it show in the README, which also shows in the description for the repo. Thoughts?

**Review by julio-lopez — APPROVED** [review]

**Pull request merged**